polygon/bridge/bridge.go

```diff
@@ -151,13 +146,13 @@ func (b *Bridge) Unwind(ctx context.Context, tip *types.Header) error {
 }
 
 // GetEvents returns all sync events at blockNum
-func (b *Bridge) GetEvents(ctx context.Context, blockNum uint64) ([]*types.Message, error) {
+func (b *Bridge) GetEvents(ctx context.Context, blockNum uint64) ([][]byte, error) {
```

Comment:
can we keep this as `[]*types.Message, error`?
as we discussed we can use `msg.Data()` and `msg.To()`